mobile/src/src/components/Housing/Listings/ListingsContainer.jsx

```diff
@@ -42,18 +42,53 @@ class ListingContainer extends Component {
   };
 
   likeListing = (id, shouldLike) => {
+    // updates number of likes
+    let index = this.state.listings.findIndex((obj) => obj.id == id);
+    let copyList = JSON.parse(JSON.stringify(this.state.listings));
+    shouldLike == true
+      ? (copyList[index].likes = this.state.listings[index].likes + 1)
+      : (copyList[index].likes = this.state.listings[index].likes - 1);
+    this.setState({ listings: copyList });
+
     ListingsEndpoint.like(
       id,
       shouldLike,
       () => {},
       (err) => {
         console.log(err);
+        // undo changes
+        shouldLike == true
+          ? (copyList[index].likes = this.state.listings[index].likes - 1)
+          : (copyList[index].likes = this.state.listings[index].likes + 1);
+        this.setState({ listings: copyList });
+      }
+    );
+  };
+
+  sendComment = (data) => {
+    // updates number of comments
+    let index = this.state.listings.findIndex(
+      (obj) => obj.id == data.object_id
+    );
+    let copyList = JSON.parse(JSON.stringify(this.state.listings));
+    copyList[index].comments = this.state.listings[index].comments + 1;
+    this.setState({ listings: copyList });
+
+    CommentsEndpoint.post(
```

Comment:
Not entirely necessary, but you could remove the need to send the contentType prop if you locked in data.content_type in this function, as it's going to remain constant. If, in your opinion, this would hurt readability, another alternative is to explicitly add content_type as a parameter of the CommentsEndpoint.post function, and give it the constant value of `Choices.contentTypes.listing`  when you call the function here. Idk though, it's up to you.